department-of-veterans-affairs/caseflow
# Removes New Special Issues From Legacy Appeals - Stage 1

Connects #14714 
Reverts #14407 - code portion

### Description
Reverts the addition of the new contentions to the legacy appeal object except the actual dropping of the columns, which will follow.

### Acceptance Criteria

### Testing Plan
1. Sign in as a Caseflow Dispatch User & work through Establishing a claim. You should not get an error, nor see any of the new contentions nor validation


1. Sign in as a Judge or an Attorney. Run through case checkout and confirm the new contentions appear and are selectable.
